packages/docs/src/components/header/components/LanguageDropdown.vue

```diff
@@ -78,11 +78,11 @@ export default defineComponent({
 
 <style lang="scss">
 .language-dropdown {
-  font-weight: 700;
+  font-weight: 600;
```

Comment:
Button style will overwrite this, that's why we could just rely on `.va-button__content` and remove this, thanks.

Reply:
> * [ ]   increase the default p font size to 1.2rem (not 1.1 rem)
> 
> Please tell us if you want us to finish this.

Hm, i thought i closed this item, will check & finish at monday.

Reply:
Default p font-size was already increased, unnessesary font-weight was removed.